src/api/app/components/notification_notifiable_link_component.rb

```diff
@@ -49,17 +50,18 @@ def notifiable_link_text
     when 'Event::ClearedDecision'
       # All reports should point to the same reportable. We will take care of that here:
       # https://trello.com/c/xrjOZGa7/45-ensure-all-reports-of-a-decision-point-to-the-same-reportable
-      report = @notification.notifiable.reports.first
-      "Cleared #{report.reportable.class.name} Report"
+      # This reportable won't be nil once we fix this: https://trello.com/c/vPDiLjIQ/66-prevent-the-creation-of-reports-without-reportable
+      "Cleared #{@notification.notifiable.reports.first.reportable&.class&.name} Report".squish
```

Comment:
If there is no reportable, the string becomes "Cleared Report" thanks to `squish` collapsing all spaces into just one